### PR TITLE
initial commit of getting started migration

### PR DESCRIPTION
**Pull request by jhadobe**
## Purpose of this pull request

This pull request (PR) ...

## Affected pages

-  ...

## Links to related PRs or Jira tickets

-  ...

## Checklist:

-  [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
-  [ ] My changes follow the code style of this project.
-  [ ] I have read the **CONTRIBUTING** document.
-  [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by jhadobe** [comment]
need to update main index page

**Review by keharper — COMMENTED** [review]
Round 1 comments. Posting these so you can begin work until I finish adding all my comments.

File: src/pages/get-started/create-integration.md [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+
+   ```bash
+   bin/magento setup:upgrade
+   ```
+
+1. Run the following command to generate the new code.
+
+    <inlinealert variant="info" slots="text"/>
+
+   In Production mode, you may receive a message to 'Please rerun Magento compile command'.  Enter the command below. Magento does not prompt you to run the compile command in Developer mode.
```
Comment (keharper) [inline src/pages/get-started/create-integration.md:274]:
```suggestion
   In Production mode, you may receive a message to 'Please rerun Magento compile command'.  Enter the command below. You are not prompted to run the compile command in Developer mode.
```

File: src/pages/get-started/create-integration.md [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+
+### Login page
+
+When a merchant clicks the **Activate** button in Admin, a pop-up login page for the third-party application displays. Magento sends values for `oauth_consumer_key` and `success_call_back` parameters. The application must store the value for `oauth_consumer_key` to tie it to the login ID. Use the `success_call_back` parameter to return control back to Magento.
+
+### Callback page
+
+The callback page must be able to perform the following tasks:
+
+*  Receive an initial HTTPS POST that Magento sends when the merchant activates integration. This post contains the Magento store URL, an `oauth_verifier`, the OAuth consumer key, and the OAuth consumer secret. The consumer key and secret are generated when the integration is created.
```
Comment (keharper) [inline src/pages/get-started/create-integration.md:306]:
```suggestion
*  Receive an initial HTTPS POST that Commerce sends when the merchant activates integration. This post contains the Commerce store URL, an `oauth_verifier`, the OAuth consumer key, and the OAuth consumer secret. The consumer key and secret are generated when the integration is created.
```

File: src/pages/get-started/create-integration.md [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+
+*  Ask for a request token. A request token is a temporary token that the user exchanges for an access token. Use the following API to get a request token from Magento:
+
+   `POST /oauth/token/request`
+
+   See [Get a request token](./authentication/gs-authentication-oauth.md#get-a-request-token) for more details about this call.
+
+*  Parse the request token response. The response contains an `oauth_token` and `oauth_token_secret`.
+
+*  Ask for an access token. The request token must be exchanged for an access token. Use the following API to get a request token from Magento:
```
Comment (keharper) [inline src/pages/get-started/create-integration.md:316]:
```suggestion
*  Ask for an access token. The request token must be exchanged for an access token. Use the following API to get a request token:
```

File: src/pages/get-started/create-integration.md [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
+
+   `POST /oauth/token/access`
+
+   See [Get an access token](./authentication/gs-authentication-oauth.md#get-an-access-token/) for more details about this call.
+
+*  Parse the access token response. The response contains an `oauth_token` and `oauth_token_secret`. These values will be different than those provided in the request token response.
+
+*  Save the access token and other OAuth parameters. The access token and OAuth parameters must be specified in the `Authorization` header in each call to Magento.
```
Comment (keharper) [inline src/pages/get-started/create-integration.md:324]:
```suggestion
*  Save the access token and other OAuth parameters. The access token and OAuth parameters must be specified in the `Authorization` header in each call to Commerce.
```

File: src/pages/get-started/create-integration.md [hunk 5]
```diff
@@ -0,0 +1,332 @@
...
+
+## Related Topics
+
+*  [Web API authentication](./authentication/)
+*  [OAuth-based authentication](./authentication/gs-authentication-oauth.md)
+*  [Magento System Requirements](https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements.html)
```
Comment (keharper) [inline src/pages/get-started/create-integration.md:330]:
```suggestion
*  [System Requirements](https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements.html)
```

File: src/pages/get-started/web-api-functional-testing.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
+---
+title: Web API Functional Testing | Getting Started with Web APIs
+description: How to implement and run functional tests for your Magento API.
+---
+
+# Web API Functional Testing
+
+The Web [API](https://glossary.magento.com/api) testing framework allows you to test Magento [Web API](https://glossary.magento.com/web-api) from the client application point of view. The tests can be used with either REST or SOAP. The REST or SOAP [adapter](https://glossary.magento.com/adapter) that runs the tests is specified in PHPUnit configuration. See [How to Run the Tests](#how-to-run-the-tests) for more information.
```
Comment (keharper) [inline src/pages/get-started/web-api-functional-testing.md:8]:
```suggestion
The Web [API](https://glossary.magento.com/api) testing framework allows you to test [Web APIs](https://glossary.magento.com/web-api) from the client application point of view. The tests can be used with either REST or SOAP. The REST or SOAP [adapter](https://glossary.magento.com/adapter) that runs the tests is specified in PHPUnit configuration. See [How to Run the Tests](#how-to-run-the-tests) for more information.
```

File: src/pages/get-started/web-api-functional-testing.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
+---
+title: Web API Functional Testing | Getting Started with Web APIs
+description: How to implement and run functional tests for your Magento API.
```
Comment (keharper) [inline src/pages/get-started/web-api-functional-testing.md:3]:
```suggestion
description: How to implement and run functional tests on Web APIs.
```

File: src/pages/get-started/web-api-functional-testing.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+
+   `extension=php_soap.dll`
+
+1. Before running the functional tests you need to clear your [cache](https://glossary.magento.com/cache). Now you are ready to run the tests.
+
+### Running the Tests
+
+1. Copy `dev/tests/api-functional/phpunit_rest.xml.dist` and `phpunit_soap.xml.dist` to `dev/tests/api-functional/phpunit_rest.xml` and `phpunit_soap.xml`.
+
+1. Define the Magento instance URL as a value of `TESTS_BASE_URL`, Test Webservice User as value of `TESTS_WEBSERVICE_USER` and Test Webservice API key as value of `TESTS_WEBSERVICE_APIKEY` in copied file i.e. `phpunit_rest.xml` or `phpunit_soap.xml`.
```
Comment (keharper) [inline src/pages/get-started/web-api-functional-testing.md:107]:
```suggestion
1. Define the Commerce instance URL as a value of `TESTS_BASE_URL`, Test Webservice User as value of `TESTS_WEBSERVICE_USER` and Test Webservice API key as value of `TESTS_WEBSERVICE_APIKEY` in copied file i.e. `phpunit_rest.xml` or `phpunit_soap.xml`.
```

File: src/pages/get-started/web-api-functional-testing.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+
+1. Copy `dev/tests/api-functional/config/install-config-mysql.php.dist` to `dev/tests/api-functional/config/install-config-mysql.php`.
+
+1. Configure your DB connection and install settings in `dev/tests/api-functional/config/install-config-mysql.php`. Specify the Magento database. The base URL to access this Magento instance must be the same specified in the `phpunit_rest.xml` or `phpunit_soap.xml` file.
```
Comment (keharper) [inline src/pages/get-started/web-api-functional-testing.md:111]:
```suggestion
1. Configure your DB connection and install settings in `dev/tests/api-functional/config/install-config-mysql.php`. Specify the Commerce database. The base URL to access this instance must be the same specified in the `phpunit_rest.xml` or `phpunit_soap.xml` file.
```

File: src/pages/index.md [hunk 10]
```diff
@@ -9,15 +9,14 @@ description: Learn about using REST and GraphQL to
 
 # Commerce Web APIs
 
-Lorem ipsum dolor sit amet
+REST, GraphQL, and Getting Started
```
Comment (keharper) [inline src/pages/index.md:12]: This needs to be a sentence describing the web APIs, not a list.

File: src/pages/index.md [hunk 11]
```diff
@@ -33,20 +32,20 @@ Use these sections to learn about how to use our web APIs to integrate with thir
 
 [Getting Starting](get-started/)
 
-Lorem ipsum dolor sit amet
+The Magento web API framework provides integrators and developers the means to use web services that communicate with the Magento system.
```
Comment (keharper) [inline src/pages/index.md:35]:
```suggestion
The Commerce web API framework provides integrators and developers the means to use web services that communicate with the Adobe Commerce.
```

Comment out the other discover blocks

File: src/pages/index.md [hunk 12]
```diff
@@ -33,20 +32,20 @@ Use these sections to learn about how to use our web APIs to integrate with thir
 
 [Getting Starting](get-started/)
```
Comment (keharper) [inline src/pages/index.md:33]:
```suggestion
[Getting Started](get-started/)
```

File: src/pages/get-started/gs-web-api-request.md [hunk 13]
```diff
@@ -0,0 +1,284 @@
...
+   $token = 'token';
+   $httpHeaders = new \Zend\Http\Headers();
+   $httpHeaders->addHeaders([
+      'Authorization' => 'Bearer ' . $token,
+      'Accept' => 'application/json',
+      'Content-Type' => 'application/json'
+   ]);
+   ```
+
+1. Open the [Magento/Customer/etc/webapi.xml](https://github.com/magento/magento2/tree/{{ page.guide_version }}/app/code/Magento/Customer/etc/webapi.xml)  configuration file and find the [CustomerRepositoryInterface](https://github.com/magento/magento2/tree/{{ page.guide_version }}/app/code/Magento/Customer/Api/CustomerRepositoryInterface.php) interface with the `getList` method.
```
Comment (keharper) [inline src/pages/get-started/gs-web-api-request.md:147]: invalid links

File: src/pages/get-started/authentication/index.md [hunk 14]
```diff
@@ -0,0 +1,167 @@
...
+```
+
+For example, in the preceding `webapi.xml` for the customerGroups resource, only a user with `Magento_Customer::group` authorization can `GET /V1/customerGroups/:id`. On the other hand, you can create a customer using `POST /V1/customers` anonymously (or by a guest).
+
+[Authorization](https://glossary.magento.com/authorization) is granted to either an administrator (or an integration) defined in the Admin with the customer group selected as one of the resources in the ACL tree.
+
+<InlineAlert variant="info" slots="text"/>
+
+A guest or anonymous is a special permission that doesn't need to be defined in `acl.xml` (and will not show up in the permissions tree in the Admin). It just indicates that the current resource in `webapi.xml` can be accessed without the need for authentication.
+<br/><br/>
```
Comment (keharper) [inline src/pages/get-started/authentication/index.md:91]: Note blocks on the devsite can't contain multiple paragraphs, and using  br elements doesn't work. You'll have to write around it, probably by removing the InlineArt and br elements.

File: src/pages/get-started/create-integration.md [hunk 15]
```diff
@@ -0,0 +1,332 @@
...
+
+1. Run the following command to update the [database schema](https://glossary.magento.com/database-schema) and data.
+
+   ```bash
+   bin/magento setup:upgrade
+   ```
+
+1. Run the following command to generate the new code.
+
+    <InlineAlert variant="info" slots="text"/>
```
Comment (keharper) [inline src/pages/get-started/create-integration.md:272]: This is another multiparagraph note that doesn't render. I would create an unofficial note like this: **Note:** In Production mode...

File: src/pages/index.md [hunk 16]
```diff
@@ -9,15 +9,14 @@ description: Learn about using REST and GraphQL to
...
+The Commerce Web APIs allow you to access and interact with your, integrate third-party software, and build a headless applications. The Web API documentation is divided into three guides: REST, GraphQL, and Getting Started.
 
 <Resources slots="heading, links"/>
 
 ## Resources
 
-*  [X](https://developer.adobe.com)
-*  [Y](https://developer.adobe.com)
-*  [Z](https://developer.adobe.com)
+*  [PHP Developer Guide](https://developer.adobe.com/commerce/php/development)
```
Comment (keharper) [inline src/pages/index.md:18]: Add a link to the REST API reference - https://magento.redoc.ly/2.4.4-admin/

**Review by keharper — COMMENTED** [review]
The top navigation needs to remove links to REST and GraphQL. 

I think the DiscoverBlocks on the front page can include links outside the repo. Uncomment those from the front and add descriptions/links to REST/GraphQL

File: src/pages/get-started/create-integration.md (hunk 15 above)
Comment (jhadobe) [reply src/pages/get-started/create-integration.md]: it seems we can use `<div>` tags to include multiple paragraphs.

**Review by keharper — COMMENTED** [review]
You still need to update the top nav to remove REST and GraphQL.

**Review by keharper — APPROVED** [review]

**Pull request merged**